content/release-notes/1.33.0.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 2021-03-05T10:30:14-08:00
+linktitle: "1.33.0"
+title: "1.33.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+{{<features>}}
+* NFS support in snapshots
+{{</features>}}
+
+{{<changes>}}
+* Added instrumentation for preflight usage.
+* Reduced disk usage during airgap install. 
+* Upgraded [Helm v3](/vendor/helm/using-helm-charts/) dependency from v3.2.4 to v3.5.2
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug where boolean custom fields were not displayed properly on the [license page](/kotsadm/updating/license-updates/).
+* Show correct page title while on the [Cluster Management](/vendor/guides/ha-guide/#adding-remaining-nodes-to-cluster) page. 
```

Comment:
This should probably follow the "Fixed a bug..." format.

`Fixed a bug where the wrong page title was shown on the Cluster Management page`